sdk/cosmos/azure-spring-data-cosmos/pom.xml

```diff
@@ -205,6 +204,8 @@
                 <include>org.json:json:[20140107]</include> <!-- {x-include-update;org.json:json;external_dependency} -->
                 <include>com.fasterxml.jackson.module:jackson-module-parameter-names:[2.10.0]</include> <!-- {x-include-update;com.fasterxml.jackson.module:jackson-module-parameter-names;external_dependency} -->
                 <include>javax.annotation:javax.annotation-api:[1.3.2]</include> <!-- {x-include-update;javax.annotation:javax.annotation-api;external_dependency} -->
+                <include>org.apache.commons:commons-lang3:[3.8.1]</include> <!-- {x-include-update;org.apache.commons:commons-lang3;external_dependency} -->
+                <include>com.google.code.findbugs:jsr305:[3.0.2]</include> <!-- {x-include-update;com.google.code.findbugs:jsr305;external_dependency} -->
```

Comment:
You shouldn't need com.google.code.findbugs:jsr305 in the bannedDependency includes. This particular entry was removed from this file as part of the same [PR](https://github.com/Azure/azure-sdk-for-java/pull/13211) that fixed in the inheritance.

Reply:
Sure @JimSuplizio - will get rid of it from bannedDependencies includes.